db/migrate/20150624133703_devise_create_users.rb

```diff
@@ -2,6 +2,7 @@ class DeviseCreateUsers < ActiveRecord::Migration
   def change
     create_table(:users) do |t|
       ## Database authenticatable
+      t.string :name
```

Comment:
This needs to be added in a new migration since we've moved past the initial devise set up.